docs/src/modules/java/pages/actions-publishing-subscribing.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+== Publishing Entity Events to a Topic
+
+To illustrate how to publish entity events, we will assume the existence of an xref:eventsourced.adoc[Event Sourced] Counter entity that emits events of types: `ValueIncreased`, `ValueDecreased` and `ValueReset`. We will get the events delivered to an Action, apply some transformation and let them be published to a https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] topic. 
+
+
+[source,protobuf,indent=0]
+.src/main/proto/actions/counter_pubsub.proto
+----
+include::example$java-eventsourced-counter/src/main/proto/actions/counter_pubsub.proto[tag=counter-pubsub]
+----
+<1> Import the Counter Domain file containing the definitions of the events. This is typically a proto definition within the same service.
+<2> The `Increase` method receives the event `ValueIncreased` and returns the transformed type `Increased`. 
+<3> The `(akkaserverless.method).eventing.in` annotation indicates that events from the entity type `counter` should be delivered to this method (when the type is `ValueIncreased`).
+<4> The `(akkaserverless.method).eventing.out` annotation indicates that the returned value from this method should be published to the topic called `counter-events`.
+<5> We do the same for the `ValueDecreased`.
+<6> Finally, we have a last method called `IgnoreReset` annotated with `(akkaserverless.method).eventing.in`, but not with `(akkaserverless.method).eventing.out`. This mehtod recieves the `ValueReset` and `Empty`. Therefore, events of type `ValueReset` will be handled, but not published to the topic.
+
+
+=== Implementing the Action
+
+The class `CounterTopicAction` gets generated for us based on the proto file defined above.
+
+[source,protobuf,indent=0]
+.src/main/java/com/example/actions/CounterTopicAction.java
+----
+include::example$java-eventsourced-counter/src/main/java/com/example/actions/CounterTopicAction.java[tag=counter-pubsub]
+----
+<1> We convert the incoming domain event `CounterDomain.ValueIncreased` to the outgoing topic API `CounterTopicApi.Increased`.
+<2> We can access the id of the originating entity throught the event subject metadata.
+<3> We use the converted object to reply.
+<4> Similar code should be written for the decrease method. Omitted here for conciseness. 
+<5> Finally, the 'ignoreReset' method is implemented with `effects().noReply()` signaling that it's a noops call.
+
+=== Subscribing and acting
+
+In example above, an Action is used to consume events from the journal, transform them to some external format and publish them to a topic. Another possible usage for Actions is to consume events and act upon. For example, you may consume events from one entity, transform to commands and send to an another entity or an external system. For that purpose, it's enough to add the `(akkaserverless.method).eventing.in` and omit the `(akkaserverless.method).eventing.out`, like we did with the `ignoreReset` method.
```

Comment:
Not sure if we need a concrete example here. Maybe we can link to the forward example.